azext_iot/monitor/parsers/common_parser.py

```diff
@@ -27,6 +28,7 @@ def __init__(self, message: Message, common_parser_args: CommonParserArguments):
         self._message = message
         self.device_id = self._parse_device_id(message)
         self.interface_name = self._parse_interface_name(message)
+        self.module_id = self._parse_module_id(message)
```

Comment:
Would it be possible to add a quick UT for parsing `model_id`?

Reply:
#wontfix

Reply:
This is something I'd like to insist on, any parsing issues related to `model_id` will crash our event monitor, and the crash will likely be silent due to the way we are suppressing output